packages/contracts/mud.config.mts

```diff
@@ -19,6 +19,13 @@ const config = {
       },
     },
   },
+  modules: [
+    {
+      name: "KeysWithValueModule",
+      root: true,
+      args: [resolveTableId("CounterTable")],
```

Comment:
is this basically a pointer to the `CounterTable` definition in `tables`?

if so `resolveTableId` sounds more like an internal operation, not something that a user would recognize as a "pointer" or "reference", so maybe we rename this to something like `tableReference` or `tableFromConfig` or `configValue('tables.CounterTable')`?

not sure how best to represent the config referencing itself haha